service/dap/server.go

```diff
@@ -735,9 +736,21 @@ func (s *Server) setClientCapabilities(args dap.InitializeRequestArguments) {
 	s.clientCapabilities.supportsVariableType = args.SupportsVariableType
 }
 
-// Default output file pathname for the compiled binary in debug or test modes,
-// relative to the current working directory of the server.
-const defaultDebugBinary string = "./__debug_bin"
+func tempDebugBinary() (string, error) {
```

Comment:
Instead of returning the error just log it and return `"__debug_bin"`

Reply:
Done.